core/server/data/exporter/index.js

```diff
@@ -6,7 +6,7 @@ var _ = require('lodash'),
     common = require('../../lib/common'),
     security = require('../../lib/security'),
     models = require('../../models'),
-    EXCLUDED_TABLES = ['accesstokens', 'refreshtokens', 'clients', 'client_trusted_domains'],
+    EXCLUDED_TABLES = ['accesstokens', 'refreshtokens', 'clients', 'client_trusted_domains', 'sessions'],
```

Comment:
@allouis What about api keys? Should they be excluded?

And in general: what about importing/exporting integerations?

Reply:
> Is it mainly used for transferring a blog to another server?

Yeah and backup for the same blog.

> If so, you might want all your integrations to keep working? 

Yeah. Just need to think about the security risk of exporting admin keys.